packages/server/graphql/mutations/archiveTeam.ts

```diff
@@ -10,10 +10,14 @@ import publish from '../../utils/publish'
 import segmentIo from '../../utils/segmentIo'
 import standardError from '../../utils/standardError'
 import {GQLContext} from '../graphql'
-import ArchiveTeamPayload from '../types/ArchiveTeamPayload'
 
 export default {
-  type: new GraphQLNonNull(ArchiveTeamPayload),
+  type: new GraphQLNonNull(
+    new GraphQLObjectType({
```

Comment:
This is the only novel pattern in the whole PR.
Since `ArchiveTeamPayload` is created using the SDL, it's not available to the legacy JS objects.
To use it, we just created a stub  inline & that stub will get filled in when the SDL gets merged into schema.
This is useful for things that we're not ready to migrate yet.
For example, in this PR I wanted to migrate the Notification types, their corresponding payload types, but not the mutations themselves.